tsconfig.json

```diff
@@ -20,6 +20,10 @@
         "paths": {
             "~/*": [
                 "src/*"
+            ],
```

Comment:
Was this not removed explicit in an other PR. Or do i remember wrong?

Reply:
It may have been, we'll roll this change back?

Reply:
It is required until https://github.com/shripalsoni04/nativescript-webview-interface/pull/44 has been merged into master

Reply:
I think we should roll this back 😄